slack.go

```diff
@@ -24,60 +38,30 @@ func NewBot(token string) *Bot {
 	}
 }
 
-func (b *Bot) Start(manager JobManager) error {
-	slack := slacker.NewClient(b.token)
-
-	manager.SetNotifier(b.jobResponder(slack))
-
-	slack.DefaultCommand(func(request slacker.Request, response slacker.ResponseWriter) {
+func (b *Bot) initialize(client *slacker.Slacker, manager JobManager) {
```

Comment:
Would recommend you name this `slack` instead of `client` - it's not really a client, it's a server, and it's confusing to call it `client`.

Reply:
Done!